src/styles.css

```
@@ -131,4 +229,560 @@
+  }
+
+  .prose pre code {
+    display: block;
+    width: fit-content;
+    min-width: 100%;
+  }
+
+  .prose img,
+  .prose video {
+    @apply rounded-md;
+    max-width: 100%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)